api/goldens/aptos_api__tests__transactions_test__test_get_transactions_output_user_transaction_with_entry_function_payload.json

```diff
@@ -114,7 +114,7 @@
     "state_change_hash": "",
     "event_root_hash": "",
     "state_checkpoint_hash": null,
-    "gas_used": "6",
+    "gas_used": "8",
```

Comment:
A noob question: Event V2 (module event) increase gas usage? If so, what is the cause?

Reply:
additional i think